airbyte-integrations/connectors/source-mailchimp/source_mailchimp/source.py

```diff
@@ -64,10 +64,12 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
     def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         authenticator = MailChimpAuthenticator().get_auth(config)
         campaign_id = config.get("campaign_id")
+        list_id = config.get("list_id")
```

Comment:
If list_id is obtained from the config, shouldn't the spec.yaml file be updated? Or at least some indication on where to set this value when setting up the connector.

Reply:
I believe that should be removed. My intention is to get the "list_id" from the parent stream "Lists".